libs/dtm/dtm_recovery.py

```diff
@@ -134,14 +139,17 @@ def perform_ops(self, workload_info: list, queue, skipread: bool = True,
             queue.put([False, f"S3bench workload for failed."
                               f" Please read log file {log_path}"])
 
-    def process_restart(self, master_node, pod_prefix, container_prefix, process,
-                        recover_time: int = 30):
+    # pylint: disable-msg=too-many-locals
+    def process_restart(self, master_node, health_obj, pod_prefix, container_prefix, process,
+                        process_ids: list = None, recover_time: int = 30):
         """
         Restart specified Process of specific pod and container
         :param master_node: Master node object
+        :param health_obj: Master node health object
         :param pod_prefix: Pod Prefix
         :param container_prefix: Container Prefix
         :param process: Process to be restarted.
+        :param process_ids: List of Process IDs
```

Comment:
we can remove these process_ids now ?

Reply:
No, we will need process ids if we want to check status for specific processes. It is an optional parameter, will mention the same.